GNUmakefile.main

```diff
@@ -0,0 +1,113 @@
+BRANCH := main
```

Comment:
VSCode does not automatically recognize this file as a make file 😢 
How about adding .mk suffix?

Reply:
do: https://github.com/vulsio/vuls-data-db/compare/1a8a8b5e9555517b762e1f66ea94d4ba88015af7..4dc5f075251df01ca84745b74f7823cbb1de3379